docpages/example_programs/interactions_and_components/commandhandler.md

```diff
@@ -48,7 +48,7 @@ int main()
 			/* Command description */
 			"A test ping command",
 
-			/* Guild id (omit for a global command) */
```

Comment:
this change is incorrect-guild id needs to be there for guild commands, but not for global commands